t/app/script/archive_old_enquiries.t

```diff
@@ -505,4 +506,118 @@ subtest 'date based updates do not happen without commit' => sub {
     };
 };
 
+my $archive_reports = sub {
+    my $opts = shift;
+
+    my ($report) = $mech->create_problems_for_body(1, $oxfordshire->id, 'Test', {
+        areas      => ',2237,',
+        user_id    => $user->id,
+    });
+
+    my ($report1) = $mech->create_problems_for_body(1, $oxfordshire->id . "," .$west_oxon->id, 'Test', {
+        areas      => ',2237,',
+        lastupdate => '2015-12-01 07:00:00',
+        user       => $user,
+    });
+
+    my ($report2) = $mech->create_problems_for_body(1, $oxfordshire->id, 'Test 2', {
+        areas      => ',2237,',
+        lastupdate => '2015-12-01 08:00:00',
+        user       => $user,
+        state      => 'investigating',
+    });
+
+    my ($report3, $report4) = $mech->create_problems_for_body(2, $oxfordshire->id, 'Test', {
+        areas      => ',2237,',
+        lastupdate => '2014-12-01 07:00:00',
+        user       => $user,
+    });
+
+    my ($report5) = $mech->create_problems_for_body(1, $oxfordshire->id . "," .$west_oxon->id, 'Test', {
+        areas      => ',2237,',
+        lastupdate => '2014-12-01 07:00:00',
+        user       => $user,
+        state      => 'in progress'
+    });
+
+    FixMyStreet::Script::ArchiveOldEnquiries::archive($opts);
+
+    $report->discard_changes;
```

Comment:
I thought this was necessary to fetch the new values from the DB (by analogy with other code in the same test, but maybe I misinterpreted it).

Reply:
It would be if you were wanting to look at something in `$report`, but as I said, you don't actually then look at `$report` in any way, so this line is superfluous.